pkg/operation/botanist/component/etcd/monitoring.go

```diff
@@ -115,6 +115,18 @@ const (
         within the last hour.
       summary: High number of failed etcd proposals
 
+  - alert: KubeEtcd3HighMemoryConsumption
+    expr: sum(container_memory_working_set_bytes{pod="etcd-main-0",container="etcd"}) / sum(vpa_spec_container_resource_policy_allowed{allowed="max",container="etcd", targetName="etcd-main", resource="memory"}) > .4
```

Comment:
Does the `vpa_spec_container_resource_policy_allowed` part work when there is no VPA enabled for the Seed - for example the VPA in the Seed spec is not enabled and there is no other VPA for the Seed?

Reply:
In our setup we always enable VPA for Seeds, but I think Seed without VPA should be still valid and supported scenario. @rfranzke do we have the hard requirement to have VPA always enabled for the Seed?

Reply:
@ialidzhikov The requirement is that a VPA must run for the seed. If the seed itself is a shoot then it can be enabled via the shoot spec, otherwise, it can run in the `garden` namespace of the seed itself.

Reply:
Do we have this requirement documented somewhere?

Reply:
We only have https://github.com/gardener/gardener/blob/master/docs/usage/seed_settings.md#vertical-pod-autoscaler, I believe.

Reply:
Ah, okay, thanks for sharing.